### PR TITLE
Feature/sidemenus

### PR DESCRIPTION
**Pull request by dayhaysoos**
right menu for filtering set up. Needs to connect with data.

### CODE REVIEW

File: src/components/FilterMenu/FilterMenu.js [hunk 1]
```diff
@@ -1,109 +1,101 @@
...
-          underlayColor="white"
-        />
-        <Accordion
-          header={<Text style={Styles.header}>{speedFilters.category}</Text>}
-          content={<View style={Styles.expanded}>{this.filterRender(speedFilters)}</View>}
+  accordionRender(allFilters) {
+    return Object.keys(allFilters).map((f, key) => {
+      this.filterRender(allFilters[f])
+      return (
+        <FilterAccordian
```
Comment (scottdj92) [inline src/components/FilterMenu/FilterMenu.js:54]: Accordion

**Comment by scottdj92** [comment]
Got a single comment, very minor

File: src/components/Toolbar/Toolbar.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import {
   TouchableHighlight
 } from 'react-native';
 
-import SearchBar from '../Searchbar/Searchbar';
+import SearchBar from '../SearchBar/SearchBar';
```
Comment (scottdj92) [inline src/components/Toolbar/Toolbar.js:11]: is this `SearchBar` or `Searchbar`?

Comment (dayhaysoos) [reply src/components/Toolbar/Toolbar.js:11]: SearchBar

**Pull request merged**